aten/src/ATen/native/native_functions.yaml

```diff
@@ -1586,6 +1586,7 @@
   dispatch:
     CPU: log_softmax_cpu
     CUDA: log_softmax_cuda
+    HammerBlade: log_softmax_hb
```

Comment:
Exactly what I need! Sweet!

Reply:
Yep, almost every modern classification network uses this as the last layer.

Reply:
Have you looked at any loss function yet? I'm trying to get `CrossEntropyLoss` to work. It somehow uses this logSoftMax kernel

Reply:
No, haven't looked at loss functions yet. Those are definitely needed for training!